restapi/src/main/java/io/stargate/web/docsapi/exception/ErrorCode.java

```diff
@@ -50,9 +49,7 @@
 
   DOCS_API_GENERAL_INVALID_FIELD_NAME(
       Response.Status.BAD_REQUEST,
-      String.format(
-          "The characters %s are not permitted in JSON field names.",
-          DocumentDB.getForbiddenCharactersMessage())),
+      "Array paths with brackets are not permitted in JSON field names."),
```

Comment:
The message appears to be too far away from the meaning that can be deduced from the name of  the constant: `DOCS_API_GENERAL_INVALID_FIELD_NAME` :thinking:

Reply:
Yea let's refactor the error code here or?